### DISCUSSION TITLE
Installazione Home Assistant

### DISCUSSION THREAD

**Discussion by gericom90** [indomus/forum#389] [HUB: Home Assistant]
Buona giornata a tutti,
sto seguendo le guide per l'installazione di Home Assistant in Docker su Raspberry .
Ho un dubbio in quanto mi trovo due file configuration.yaml;
nei seguenti path:
/home/pi/homeassistant/homeassistant
/home/homeassistant/.homeassistant
Ho forse incrociato due installazioni diverse ?
Mi sapete inoltre indicare un buon manuale (possibilmente base) per imparare la programmazione su Raspberry?

Grazie
Stefano

**Comment by Scialla** [comment]
la prima è tendenzialmente sbagliata ma, se non dici che versione di HA stai installando è difficile capire....
Anche il termine programmazione su Rasp è tendenzialmente sbagliato, il Rasp è un mini computer, che intendi? E come se domandassi un manuale per imparare la programmazione di un PC... 
Cmq trovi tutte le informazioni possibili online, per qualsiasi cosa tu voglia fare, basta cercare le cose giuste.

**Reply by gericom90** [reply]
hai certamente ragione, ma essendo un ignorante in materia, mi sembra limitativo copiare ed incolare in Home Assistant i comandi (sudo, compose, ecc...) richiamati nelle indicazioni delle guide di installazione senza sapere a cosa realmente servono.
Certamente su Internet si trova di tutto, ma ritengo che chiedere a chi ne sa più di te, serva ad arrivare prima ad una soluzione al problema posto. Capisco che Raspberry è un mini computer, ma la domanda ( forse posta male) è a che linquaggio si appoggia l'applicazione Home Assistant.

**Reply by Scialla** [reply]
Home Assistant è scritta in python ma non credo interessi un granché a meno che non ti metti a scrivere script, custom component etc. I file di configurazione invece sono in YAML e non credo esista un manuale, devi solo fare prove e capire, non è così difficile. ci sono siti di yamlinter per vedere se hai fatto errori che possono aiutare, oppure Visual Studio Code che, con il plugin di HomeAssistant, aiuta moltissimo, poi dovrai studiarti i template e progredire.. Non c'è un sito che parte dalle basi e ti segue passo passo

**Comment by indomus** [comment]
Stai seguendo [questa](https://indomus.it/guide/come-installare-e-configurare-home-assistant-core-con-docker-su-un-raspberry-pi-gia-in-uso/)? Perché se sì, hai cannato i path.

**Reply by gericom90** [reply]
In effetti è proprio quella. 
Da quello che avevo capito l'installazione di Home Assistant  (seguendo i vari passaggi della guida) avrebbe dovuto portare ad una installazione su una sola Directory. Ed è quello che non capisco, perchè in realtà mi trovo con due directory diverse. 
E se provassi a cancellare l'intera Directory in più ? 
La cosa strana è che lanciando http://INDIRIZZO_IP_DI_HOME_ASSISTANT:8123  il programma Home Assistant funziona ( ma non capisco se a funzionare è quello installato in /home/pi/homeassistant/homeassistant  o in  /home/homeassistant/.homeassistant). 
Poco male, nel caso reinstallo il tutto (trattandosi di una installazione di prova per capire come funziona un sistema di H.A).
Comunque grazie ancora.

**Reply by Scialla** [reply]
Se hai seguito la guida per installare con docker, funzionerà all'interno del contenitore, dal percorso indicato si prenderà solo la configurazione, molto probabilmente la seconda che hai indicato, l'altra sarà un errore che hai commesso
